lifecycle/lifecycle-common/gradle.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2024 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+artifactRedirecting.publication.targetNames=jvm,macosX64,macosArm64,uikitX64,uikitArm64,uikitSimArm64,linuxX64
```

Comment:
uikit doesn't work here, it should be `ios`

Reply:
fixed